### PR TITLE
Updating PartsUnlimitedWebsite from aspnet5-beta8 to aspnet-rc1

### PR DESCRIPTION
**Pull request by EMaher**
Updated PartsUnlimited website to the latest stable builds of ASP.NET 5.

### CODE REVIEW

File: src/PartsUnlimitedWebsite/Startup.cs [hunk 1]
```diff
@@ -58,7 +55,7 @@ public void ConfigureServices(IServiceCollection services)
             }
             else
             {
-                services.AddEntityFramework()
+                services.AddEntityFramework()
```
Comment (twsouthwick) [inline src/PartsUnlimitedWebsite/Startup.cs]: Did you mean to add extra whitespace?

File: src/PartsUnlimitedWebsite/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -1,168 +1,168 @@
...
+    @Html.Script("bootstrap")
+    @Html.Script("bootstrap-switch")
+    @Html.Script("signalr")
+    @Html.Script("signalr-hubs")
+    @Html.Script("site")
+    @Html.Script("respond")
+
+    @RenderSection("scripts", required: false)
+</body>
+</html>
```
Comment (twsouthwick) [inline src/PartsUnlimitedWebsite/Views/Shared/_Layout.cshtml]: Did you change the encoding or newlines of this file? Why is it showing deleted/added for every line?

**Comment by twsouthwick** [comment]
A couple of comments, but LGTM

File: src/PartsUnlimitedWebsite/Startup.cs (hunk 1 above)
Comment (EMaher) [inline src/PartsUnlimitedWebsite/Startup.cs]: No.  Extra whitespace deleted.

**Pull request merged**